carapace-server/src/test/java/org/carapaceproxy/backends/UnreachableBackendTest.java

```diff
@@ -227,4 +233,82 @@ public void testNonHttpResponseThenClose() throws Exception {
         }
     }
 
+    @Test
+    public void testStuckRequest() throws Exception {
+
+        final boolean backendsUnreachableOnStuckRequests = variant;
```

Comment:
in the other methods 'variant' was 'useCache'
please create another class or use a couple of variables